### PR TITLE
feat: first draft of 'getting started' doc

### PR DESCRIPTION
**Pull request by phantomas1234**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
...
+
+The side panel on the left provides the user with additional information (for example the medium composition) and the ability to select different model simulation methods. The default simulation method, [pFBA](http://cobramethods.wikidot.com/pfba) (_parsimonious flux balance analysis_), will compute a flux distribution that maximizes growth yield and minimizes the cost of enzyme expression simultaneously. Descriptions of the other available methods will follow shortly.
+
+<img src="gifs/side-panel-medium-simulation-methods.gif" alt="Reaction knockout" width="650">
+
+Computationally more extensive methods can be chosen to get uncertainty estimates for the predicted fluxes. For example, pFBA-FVA combines [pFBA](http://cobramethods.wikidot.com/pfba) with [FVA](http://cobramethods.wikidot.com/flux-variability-analysis) (_flux variability analysis_). Uncertain flux are highlighted using transparency.
+
+<img src="gifs/pfba-fva.gif" alt="Reaction knockout" width="650">
+
+The side panel furthermore provides a selection of different pathway maps the user can explore other parts of metabolism for example (amino acid biosynthesis).
```
Comment (kvikshaug) [inline README.md]: Odd sentence, a word missing?

File: README.md [hunk 2]
```diff
@@ -1 +1,79 @@
...
\ No newline at end of file
+# Getting started
+
+Welcome to our cell factory design and analysis platform! We develop the platform as part of the [DD-DeCaF](http://dd-decaf.eu) project with the goal to put model-guided and data-driven design into practice in industrial biotechnology. The platform will enable the following key technologies:
+
+* Metagenomics-enabled design of novel enzymes and biochemical pathways.
+* Omics data-driven design of cell factories for the production of chemicals and proteins.
+* Analysis and design of microbial communities relevant to human health, industrial biotechnology and agriculture.
+
+With this interactive web application, we target primarily non-expert users that have a need to analyze omics data and compute strains designs. An advance programming interface is provided for expert users ([API docs](https://docs.dd-decaf.eu/)) enabling them deviate from the default workflows and perform custom analyses.
```
Comment (matyasfodor) [inline README.md:9]: `advance` -> `advnaced`

File: README.md [hunk 3]
```diff
@@ -1 +1,79 @@
...
+
+The platform is currently in __beta__ and tested primarily with Google Chrome. If you'd like to stay up-to-date with new releases, please [subscribe](https://tinyletter.com/dd-decaf) to our quarterly newsletter. We will be eternally grateful for feedback in case you miss features or you encountered a problem (for now please drop us a message at [niso@biosustain.dtu.dk](mailto:niso@biosustain.dtu.dk)).
+
+The following sections provide an overview of what you can currently do on platform. Have fun!
+
+### Login (optional)
+
+You can log in [here](https://app.dd-decaf.eu/login) or by navigating to
+the menu in the upper right-hand corner. We support sign-on through a number of social media platforms. Alternatively, you can contact us for an account.
```
Comment (matyasfodor) [inline README.md]: Perhaps a link for contact?

File: README.md [hunk 4]
```diff
@@ -1 +1,79 @@
...
+
+All modifications can be undone (for example the reaction deletion).
+
+<img src="gifs/undo-knockout.gif" alt="Reaction knockout" width="650">
+
+The side panel on the left provides the user with additional information (for example the medium composition) and the ability to select different model simulation methods. The default simulation method, [pFBA](http://cobramethods.wikidot.com/pfba) (_parsimonious flux balance analysis_), will compute a flux distribution that maximizes growth yield and minimizes the cost of enzyme expression simultaneously. Descriptions of the other available methods will follow shortly.
+
+<img src="gifs/side-panel-medium-simulation-methods.gif" alt="Reaction knockout" width="650">
+
+Computationally more extensive methods can be chosen to get uncertainty estimates for the predicted fluxes. For example, pFBA-FVA combines [pFBA](http://cobramethods.wikidot.com/pfba) with [FVA](http://cobramethods.wikidot.com/flux-variability-analysis) (_flux variability analysis_). Uncertain flux are highlighted using transparency.
```
Comment (matyasfodor) [inline README.md]: "Uncertain flux are" I guess flux is a tricky word, but it looks strange that it's followed by a plural verb. I just want to make sure it's intentional.

File: README.md [hunk 5]
```diff
@@ -1 +1,79 @@
...
+
+<img src="gifs/pfba-fva.gif" alt="Reaction knockout" width="650">
+
+The side panel furthermore provides a selection of different pathway maps the user can explore other parts of metabolism for example (amino acid biosynthesis).
+
+<img src="gifs/amino-acids-pathway.gif" alt="Select amino acid biosynthesis map" width="650">
+
+You can search the map by pressing Ctrl-F (Command-F on Mac). _The arrow buttons for cycling through the matches are currently not visible. This will be fixed soon._
```
Comment (matyasfodor) [inline README.md]: Isn't the `Control + F` syntax more common for hotkeys?

File: README.md [hunk 6]
```diff
@@ -1 +1,79 @@
...
+
+<img src="gifs/add-reaction.gif" alt="Add pyruvate carboxylase" width="650">
+
+The application furthermore allows the user to analyze and integrate data with models (see below for how to upload data to the platform). 
+
+__TODO: add gif for data-driven card and animation__
+
+### Computational cell factory design
+
+In the [Pathways](https://app.dd-decaf.eu/app/pathways) application you can predict metabolic routes to products that your can host can not natively produce. You can share pathways with the [Interactive Map](https://app.dd-decaf.eu/app/pathwayvis) application for further inspection.
```
Comment (matyasfodor) [inline README.md]: "your can host can" - no need for first `can` IMO.

File: README.md [hunk 7]
```diff
@@ -1 +1,79 @@
...
+
+With this interactive web application, we target primarily non-expert users that have a need to analyze omics data and compute strains designs. An advance programming interface is provided for expert users ([API docs](https://docs.dd-decaf.eu/)) enabling them deviate from the default workflows and perform custom analyses.
+
+The platform is currently in __beta__ and tested primarily with Google Chrome. If you'd like to stay up-to-date with new releases, please [subscribe](https://tinyletter.com/dd-decaf) to our quarterly newsletter. We will be eternally grateful for feedback in case you miss features or you encountered a problem (for now please drop us a message at [niso@biosustain.dtu.dk](mailto:niso@biosustain.dtu.dk)).
+
+The following sections provide an overview of what you can currently do on platform. Have fun!
+
+### Login (optional)
+
+You can log in [here](https://app.dd-decaf.eu/login) or by navigating to
```
Comment (kvikshaug) [inline README.md:17]: Suggest we use relative links so you're not e.g. unknowingly sent from staging to production

Comment (phantomas1234) [reply README.md:17]: Wouldn't that break the links if people would look at the markdown file on GitHub? Ideally, no one would ever look at staging besides us.

File: README.md (hunk 6 above)
Comment (phantomas1234) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (phantomas1234) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (phantomas1234) [reply README.md]: thanks, should indeed be plural

File: README.md (hunk 3 above)
Comment (phantomas1234) [reply README.md]: good idea

File: README.md (hunk 2 above)
Comment (phantomas1234) [reply README.md:9]: fixed

File: README.md (hunk 1 above)
Comment (phantomas1234) [reply README.md]: Changed to ```In addition to central carbon metabolism, the side panel provides an extended selection of pathway maps the user can explore. For example amino acid biosynthesis:```

**Comment by kvikshaug** [comment]
[Moved here](https://github.com/DD-DeCaF/metabolica-ui-home/pull/3)

**Pull request closed without merging**